migrations/versions/2020-01-24T02-59-46Z_b1362f9fad30_create_email_domains.py

```diff
@@ -0,0 +1,41 @@
+"""create email domains
+
+Revision ID: b1362f9fad30
+Revises:
+Create Date: 2020-01-24 02:59:46.288204+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "b1362f9fad30"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "domains",
+        sa.Column("id", sa.Integer(), nullable=False),
```

Comment:
Do we need an `id` ? IMO the domain could be used as primary key as it's unique

Reply:
I wondered the same originally, and decided in the slight chance that we need to FK to this ever, it's cleaner to use the ID as the FK than the domain (an int vs a string is semi significant in the DB). So whether or not we use the `id` it didn't seem to hurt to have. 

I think the lookup is on the domain though, so I should probably add an index there.